src/main/java/com/starrocks/connector/spark/sql/connect/StarRocksConnector.java

```diff
@@ -71,21 +77,47 @@ public static StarRocksSchema getSchema(StarRocksConfig config) {
         return new StarRocksSchema(columns, pks);
     }
 
-    private static final String TABLE_SCHEMA_QUERY =
-            "SELECT `COLUMN_NAME`, `ORDINAL_POSITION`, `COLUMN_KEY`, `DATA_TYPE`, `COLUMN_SIZE`, `DECIMAL_DIGITS` " +
-                    "FROM `information_schema`.`COLUMNS` WHERE `TABLE_SCHEMA`=? AND `TABLE_NAME`=?;";
+    public static List<String> getDatabases(StarRocksConfig config) {
+        List<Map<String, String>> dbs = extractColumnValuesBySql(config, ALL_DBS_QUERY, Arrays.asList());
+        List<String> dbNames = new ArrayList<>();
 
+        for (Map<String, String> db : dbs) {
+            String dbName = Optional.ofNullable(db.get("Database"))
+                    .orElseThrow(() -> new StarrocksException("get Database header error"));
+            dbNames.add(dbName);
+        }
 
-    // Driver name for mysql connector 5.1 which is deprecated in 8.0
-    private static final String MYSQL_51_DRIVER_NAME = "com.mysql.jdbc.Driver";
+        return dbNames;
+    }
 
-    // Driver name for mysql connector 8.0
-    private static final String MYSQL_80_DRIVER_NAME = "com.mysql.cj.jdbc.Driver";
+    public static Map<String, String> loadDatabase(StarRocksConfig config, List<String> dbNames) {
+        List<String> parameters = Arrays.asList(String.join(",", dbNames));
+        List<Map<String, String>> dbs = extractColumnValuesBySql(config, LOAD_DB_QUERY, parameters);
 
-    private static final String MYSQL_SITE_URL = "https://dev.mysql.com/downloads/connector/j/";
-    private static final String MAVEN_CENTRAL_URL = "https://repo1.maven.org/maven2/mysql/mysql-connector-java/";
+        for (Map<String, String> db : dbs) {
+            String dbName = Optional.ofNullable(db.get("SCHEMA_NAME"))
+                    .orElseThrow(() -> new StarrocksException("get Database SCHEMA_NAME error"));
+            return new DatabaseSpec(dbName).toJavaMap();
+        }
+        throw new StarrocksException("has no this database: " + config.getDatabase());
+    }
+
+    public static Map<String, String> getTables(StarRocksConfig config, List<String> dbNames) {
```

Comment:
How about use List<org.apache.spark.sql.connector.catalog.Identifier> as the return?